.github/workflows/release.yml

```
@@ -88,9 +88,60 @@ jobs:
+    name: Build ClaudeNotifier.app (macOS)
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Import Apple certificate
+        env:
+          APPLE_CERTIFICATE: ${{ secrets.APPLE_CERTIFICATE }}
+          APPLE_CERTIFICATE_PASSWORD: ${{ secrets.APPLE_CERTIFICATE_PASSWORD }}
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2046:warning:15:50: Quote this to prevent word splitting

(shellcheck)